### PR TITLE
fix: retain object form changes when choosing terms

### PR DESCRIPTION
**Pull request by lem-onade**

### CODE REVIEW

**Review by arnevandoorslaer — DISMISSED** [review]

File: packages/solid-crs-core/tsconfig.json [hunk 1]
```diff
@@ -4,8 +4,9 @@
     "baseUrl": "lib",
     "outDir": "dist",
     "emitDeclarationOnly": true,
+    "resolveJsonModule": true,
     // Recommended by Lit Element
-    "module": "es2015",
+    "module": "es2020",
     "target": "es2017",
```
Comment (BelgianNoise) [inline packages/solid-crs-core/tsconfig.json:10]: is `es2020` needed? If only 2018 is needed i would go for that.
Also remove teh comment as it does not apply anymore.

File: packages/solid-crs-manage/package.json [hunk 2]
```diff
@@ -86,10 +85,10 @@
...
     "coverageThreshold": {
       "global": {
-        "statements": 95.06,
-        "branches": 93.91,
-        "lines": 95.7,
-        "functions": 87.13
+        "statements": 91.65,
+        "branches": 87.9,
+        "lines": 92.33,
+        "functions": 81.43
```
Comment (BelgianNoise) [inline packages/solid-crs-manage/package.json:91]: I keep you in the holes eh 👀

**Review by BelgianNoise — CHANGES_REQUESTED** [review]
1. Create new object
2. choose a term for field Type
3. Form does not know it is valid or dirty -> cant save

(I showed you what is wrong in screenshare, haveing a hard time writing it out ))))

File: packages/solid-crs-core/tsconfig.json (hunk 1 above)
Comment (lem-onade) [reply packages/solid-crs-core/tsconfig.json:10]: Probably does not matter, but the `presentation` and `core` packages use it as well.

**Review by arnevandoorslaer — APPROVED** [review]

**Review by BelgianNoise — APPROVED** [review]
LGTM

**Pull request merged**